rest/admin_api_auth_test.go

```diff
@@ -845,8 +845,10 @@ func TestAdminAPIAuth(t *testing.T) {
 		},
 	}
 
+	serverIsCE := base.TestsUseServerCE()
+
 	SGWorBFArole := MobileSyncGatewayRole.RoleName
-	if base.TestsUseServerCE() {
+	if serverIsCE {
 		SGWorBFArole = BucketFullAccessRole.RoleName
 	}
```

Comment:
I wonder if we can combine this into a function to get role for sync gateway admin. Is there ever a plan to have `MobileSyncGateway` on CE server or is version of server dependent?